mirrord/layer/tests/apps/issue-1458/src/main/java/issue/Issue1458.java

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
FYI, my 2 cents - as much as I am happy to have some Java in our integration - if this becomes too hard to maintain/get working we can use Rust/Python/C (any language that lets us reproduce the case for the integration test)

Reply:
Alright. When I come back to this, will check for some github action (think mehul mentioned it), as the sample code is very self-contained (only dependency is netty + maven).

Reply:
Dropped java in favor of a simpler rust test.